parlai/zoo/model_list.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+"""This file contains a list of all the models in the model zoo, 
+the path to load them, agents & tasks associated (e.g. they were trained using) and a description.
+Using the path you should be able to download and use the model automatically, e.g.:
+ python examples/interactive.lua --model-file "models:wikipedia_2016-12-21/tfidf_retriever/drqa_docs"
+"""
+
+model_list = [
+    {
+        "id": "wikipedia_2016-12-21_retriever",
+        "path": "models:wikipedia_2016-12-21/tfidf_retriever/drqa_docs",
+        "agent":  "tfidf_retriever",
+        "task": "wikipedia_2016-12-21",
+        "description": "retrieval over Wikipedia dump, used for DrQA on the open squad dataset"
```

Comment:
Maybe mention here that it's the dump from the original DrQA paper? Just to differentiate it from the one below

Reply:
ok!